test/cmdlineTests/standard_yul_stack_opt_disabled/input.json

```diff
@@ -22,3 +28,24 @@
 	let a2, b2, c2, d2, e2, f2, g2, h2, i2, j2, k2, l2, m2, n2, o2, p2 := fun()
 	sstore(a1, a2)
 }
+"
+		}
+	},
+	"settings":
+	{
+		"optimizer": {
+			"enabled": false,
+			"details": {
+				"yul": true,
+				"yulDetails": {
+					"stackAllocation": false,
```

Comment:
The purpose of the test was that the default optimizer setting does not perform stack optimization. Maybe the test can also be fully rewritten so that it does produce code but just not optimal one?

Reply:
Like
` {let x:= 1 } { let y :=2}`
results in code that does not reallocate the same stack slot.

Reply:
I find the current one really clear in that the error message says there are 17 slots needed so it's obvious what the cause is. The fact that it's an ICE is unfortunate though.

I'd rather not rewrite it myself as I don't know enough about stack optimization yet (or I'd at least need some help). In any case I think this is better done in a separate PR so that this one does not block the one with evmasm optimizations.

Reply:
> I find the current one really clear in that the error message says there are 17 slots needed so it's obvious what the cause is. The fact that it's an ICE is unfortunate though.

But yeah, it does not test the defaults. The point of rewriting it to Standard JSON was to disable most of the optimizer though.

Reply:
> Like
> `{let x:= 1 } { let y :=2}`
> results in code that does not reallocate the same stack slot.

I just tried that and you can see the difference with `"optimizerSteps": "c"` but unfortunately with the default steps both get optimized into the same output:

Input:
```yul
{
	{
		let x := 1
		sstore(0, x)
	}
	{
		let y := 2
		sstore(32, y)
	}
}
```
Output:
```
0x01
  /* \"A\":27:28   */
0x00
  /* \"A\":20:32   */
sstore
  /* \"A\":50:51   */
0x02
  /* \"A\":61:63   */
0x20
  /* \"A\":54:67   */
sstore
```

I'm going to create an issue to fix this one because I only really wanted to quickly fix the PR and get back to the base path issue.

Reply:
Oh, looks like we'll have to fix it here after all because the ICE in Standard JSON contains absolute paths and these are different on different platforms (CI fails on macOS).

Reply:
In that case I'm switching to that simpler test but still with `"optimizerSteps": "c"`. That at least does not produce an ICE.